routes/middleware/jwt/verifyToken.ts

```diff
@@ -10,12 +10,12 @@ const verifyToken = async (req: Request, res: Response, next: NextFunction) => {
     const decoded = jwt.verify(token as string, tokenSecret).token;
```

Comment:
```suggestion
    const { pk } = jwt.verify(token as string, tokenSecret);
```
타입 지정도 해준다면 깔끔할 것 같음